packages/ndla-ui/src/model/ContentType.ts

```diff
@@ -43,27 +43,14 @@ export const RESOURCE_TYPE_ASSESSMENT_RESOURCES = "urn:resourcetype:reviewResour
 export const RESOURCE_TYPE_SOURCE_MATERIAL = "urn:resourcetype:SourceMaterial";
 export const RESOURCE_TYPE_CONCEPT = "urn:resourcetype:concept";
 
-export const ListOfContentTypes = [
-  "SUBJECT_MATERIAL",
-  "TASKS_AND_ACTIVITIES",
-  "ASSESSMENT_RESOURCES",
-  "SUBJECT",
-  "SOURCE_MATERIAL",
-  "LEARNING_PATH",
-  "TOPIC",
-  "MULTIDISIPLINARY_TOPIC",
-  "CONCEPT",
-  "EXTERNAL",
-  "MISSING",
-];
-
 export const contentTypeMapping: Record<string, string> = {
   [RESOURCE_TYPE_LEARNING_PATH]: LEARNING_PATH,
   [RESOURCE_TYPE_SUBJECT_MATERIAL]: SUBJECT_MATERIAL,
   [RESOURCE_TYPE_TASKS_AND_ACTIVITIES]: TASKS_AND_ACTIVITIES,
   [RESOURCE_TYPE_ASSESSMENT_RESOURCES]: ASSESSMENT_RESOURCES,
   [RESOURCE_TYPE_SOURCE_MATERIAL]: SOURCE_MATERIAL,
   [RESOURCE_TYPE_CONCEPT]: CONCEPT,
+  [MULTIDISCIPLINARY]: MULTIDISCIPLINARY,
```

Comment:
Må denne være med her? Dei andre er jo taksonomi-typer.

Reply:
Må er et sterkt ord. Det er den enkleste løsningen, og der jeg følte at det som strengt tatt er et emne passet best inn. Helt åpen for at vi legger det et annet sted istedenfor, eller at vi legger en plan for hvordan hele denne filen egentlig skal se ut. PDD er den litt rotete.